components/script/dom/htmliframeelement.rs

```diff
@@ -770,6 +773,8 @@ impl IframeLoadEventSteps {
 impl Runnable for IframeLoadEventSteps {
     fn handler(self: Box<IframeLoadEventSteps>) {
         let this = self.frame_element.root();
-        this.iframe_load_event_steps(self.pipeline_id);
+        if !this.cancel_initial_load_event.get() {
```

Comment:
Doesn't this cause no load events at all, since cancel_initial_load_event is never reset?